docker_build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Setup yarn and build qmk_configurator inside the docker container.
+
+set -e
+#set -x
+
+# Install node
+. /root/.nvm/nvm.sh
+nvm install 11.12
```

Comment:
you should be able to use the .nvmrc in the project for this

Reply:
Apparently that's not enough, as it complains that 11.12 is not installed when I leave that step out.